pyproject.toml

```diff
@@ -22,12 +23,14 @@ filterwarnings = [
 [tool.cibuildwheel]
 # update skip when numpy wheels become available
 skip = ["*-musllinux_*", "cp310-win32", "cp310-manylinux_i686"]
-test-requires = "pytest"
-test-command = "python -m pytest {package}/tests"
-test-skip = ["*universal2:arm64"]
+test-extras = ["test"]
+test-command = "python {package}/download_data.py && python -m pytest -vv {package}/tests"
+test-skip = ["*_arm64", "*universal2:arm64"]
```

Comment:
Since you are not building universal wheels now, I think "*universal2:arm64" can be removed?

Reply:
Yes.